python-package/geobr/lookup_muni.py

```diff
@@ -0,0 +1,71 @@
+""" Lookup municipality codes and names
+
+Input a municipality NAME or CODE and get the names and codes of the municipality's corresponding state, meso, micro,
+intermediate, and immediate regions.
+
+:param name_muni -- The municipality name to be looked up
+:param code_muni -- The municipality code to be looked up
+
+:return data.frame with 13 columns identifying the geographies information of that municipality
+
+Details Only available from 2010 Census data so far
+
+example
+
+import geobr
+
+# Lookup table for municipality of Rio de Janeiro
+
+mun = lookup_muni('Rio de Janeiro)
+or
+mun = lookup_muni(3304557)
+
+lookup table for all municipalities
+mun_all = lookup_muni("all")
+
+mun_all <- lookup_muni(code_muni = "all")
+"""
+import pandas as pd
+import utils
+
+
+def lookup_muni(name_muni='', code_muni=''):
+    # Get metadata with data url addresses
+    temp_meta = utils.download_metadata()
```

Comment:
Please use the function `utils.select_metadata`. It already filters by year, type and simplified. And handles errors and exceptions.

Reply:
Have you changed the function?